examples/chainlit/basic-app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.11
```

Comment:
i think there are stom duplicates, I still see the chainlit example duplicated:

![image](https://github.com/ploomber/doc/assets/989250/1c63df20-c3c3-4124-957c-1a3a58577e4b)

Reply:
is this because these are references in the frontend?

Reply:
@edublancas yes, I didn't do any deletions yet so there are many duplicates (some of which have references and some not). 

I wanted to go through and change the references on the frontend before deleting any duplicates, to avoid breaking anything. Once we approve this PR, I'll submit the frontend PR, and then once that is merged, I'll come back and submit a PR removing all the duplicates. I have them noted down